README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Introduction
+
+My name is Aidan Coffey and I am from Long Island, NY. I am a 4th in the College majoring in Political Science and Public Policy. (*BA Spring 2020*). 
+
+## Politcal Science Background
+
+At UChicago, I have focused my study on American politics. I have supplanted that academic focus while organizing for the Missouri Democratic Party. I recommend you check out their website here! <https://missouridemocrats.org/>
```

Comment:
Not an expert in R, so I don't know if there is only one way to do this, but is this the right way to link a url? Just commenting because it is different from how I learned to do it.

Reply:
@ahyeoncho you can, though I wouldn't generally recommend it. I think it makes more sense to embed the URL in the text itself, like this

```
At UChicago, I have focused my study on American politics. I have supplanted that academic focus while organizing for the [Missouri Democratic Party](https://missouridemocrats.org/).
```

At UChicago, I have focused my study on American politics. I have supplanted that academic focus while organizing for the [Missouri Democratic Party](https://missouridemocrats.org/).